### ISSUE TITLE
create getter to get existing column names

### ISSUE THREAD

**Issue by surchs** [neurobagel/annotation_tool#255]
- [x] when called, returns an array of strings of all column names in the provided data table (or in the data dictionary if equivalent)

Other requirements:

- [x] depending components have been checked and are calling this getter correctly
- [x] have unit tests to check functionality
- [x] needed store state variables exist already or have been created if not

**Comment by rmanaem** [comment]
@surchs Did you work on this? 
The PR that's still open refers to #254 which makes sense but the branch name has 255 in it, maybe a typo?

**Comment by surchs** [comment]
Ooh, my bad @rmanaem. I did start working on this and then I thought I should probably deal with some other stuff first and so I ended up unassigning myself and didn't open a PR. What I should have also done is uncheck the boxes here. I think after all that confusion I also used a misleading branch name for my open PR, so that's what happened... :crab: 

Feel free to take this one! I'll pay a bit more  attention to my issues in the future.
